### PR TITLE
[Doc enhancement][updates-to-in-meeting-dialog-box-notification-API][2237887]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/aa48c972090df28dc9a1cc94e419223eacb12ebd/msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/de8ede83-5120-4bea-bdc9-cdddc7abddc8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5a98130c2ced897f669a2fcb9ddc9a6f3c84b56f/msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1a4dae50-e9b3-49b7-8b7a-0d13361adca5) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/70a96a4a3d544edf20756f4621f8f75e7c2f90d1/msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1b5914ad-dc81-4e5c-a046-23419353968a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ad91a9e101ba8fc7f535a2fd67624dfba1fc5c67/msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4e736c16-744f-41f5-bd20-d724f8f842ff) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9790a77f3074ef4d0398c6d21e961a9c144ff13d/msteams-platform/apps-in-teams-meetings/API-references.md) | 92 | 24 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/640cdc24-ce1c-4c04-88c1-82c2ce91065a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9790a77f3074ef4d0398c6d21e961a9c144ff13d/msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/810cc3a5-c878-4b7d-8332-6706aedc66fd) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/298ff178c3b5fe6438a62f8d1705abf244aa35d1/msteams-platform/apps-in-teams-meetings/API-references.md) | 92 | 24 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/786ee4d2-a8cc-42b3-898b-32dbd139a071) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/298ff178c3b5fe6438a62f8d1705abf244aa35d1/msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/91f58828-c692-4191-9e88-3527ff978f82) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e8c32b0492614a0ea46d8a04007722e55c39f146/msteams-platform/apps-in-teams-meetings/API-references.md) | 92 | 23 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b39d76ad-8a92-4fd4-96c0-9a232494a9be) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e8c32b0492614a0ea46d8a04007722e55c39f146/msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/51d08f67-827c-4397-bcff-e16b72481c9f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4c9931a0bd95b815da6179e6c2018bc3ffb8f81b/msteams-platform/apps-in-teams-meetings/API-references.md) | 92 | 23 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e812a9cd-1cf8-4c36-99a0-22a673b8d88e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4c9931a0bd95b815da6179e6c2018bc3ffb8f81b/msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bd100689-d18f-4ecd-b015-93c2a2a1b3e5) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ddbb14c7ebf193e61ffe0194d272a3f9a95b1961/msteams-platform/apps-in-teams-meetings/API-references.md) | 92 | 23 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fca5323d-3a34-4ac1-830c-bf34f290aaf4) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ddbb14c7ebf193e61ffe0194d272a3f9a95b1961/msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/416c88bf-399f-458a-a6e9-211c0b49df1d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/apps-in-teams-meetings/API-references.md [hunk 1]
```diff
@@ -133,11 +133,13 @@ The following table provides the response codes:
 
 ## Send notification signal API
 
-All users in a meeting receive the notifications sent through the `NotificationSignal` API. `NotificationSignal` API enables you to provide meeting signals that are delivered using the existing conversation notification API for user-bot chat. You can send signal based on user action, an in-meeting dialog box. The API includes query parameter, examples, and response codes.
+All users in a meeting receive the notifications sent through the `NotificationSignal` API. `NotificationSignal` API triggers an in-meeting dialog box and enables you to provide meeting signals that are delivered using the existing conversation notification API for user-bot chat. You can send signal based on user action, which is an in-meeting dialog box. The API includes query parameter, examples, and response codes.
```
Comment (arocc) [inline msteams-platform/apps-in-teams-meetings/API-references.md:136]: > The API includes query parameter, examples, and response codes.

I don't think this last sentence is necessary, as the other APIs don't say this. Can replace this sentence with: "The API is available through Bot Services." to match other APIs on this doc.  Suggestion below including my other comment.

```suggestion
All users in a meeting receive the notifications sent through the `NotificationSignal` API. The `NotificationSignal` API triggers an in-meeting dialog box and enables you to provide meeting signals that are delivered using the existing conversation notification API for user-bot chat. You can send an in-meeting dialog box based on user action. The API is available through Bot Services.
```

File: msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md [hunk 2]
```diff
@@ -114,13 +114,9 @@ Messaging extension works as expected when a user is in an in-meeting view. The
...
-> * If you want your app to support anonymous users, initial invoke request payload must rely on `from.id` request metadata in `from` object, not `from.aadObjectId` request metadata. `from.id` is the user ID and `from.aadObjectId` is the Microsoft Azure Active Directory (Azure AD) ID of the user. For more information, see [using task modules in tabs](../task-modules-and-cards/task-modules/task-modules-tabs.md) and [create and send the task module](../messaging-extensions/how-to/action-commands/create-task-module.md?tabs=dotnet#the-initial-invoke-request).
+In-meeting dialog must not use task module. Task module isn't invoked in a meeting chat. An external resource URL is used to display in-meeting dialog box. You can use the `submitTask` method to submit data in a meeting chat.
```
Comment (arocc) [inline msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md:119]: After this paragraph, can you please insert a screenshot of in-meeting dialog? Similar to the meeting stage section? The screenshot in the "Desktop" section of this document would work, for example: https://docs.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/design/designing-apps-in-meetings#desktop-2

**Review by arocc — CHANGES_REQUESTED** [review]
Just a couple of suggestions for clarity!

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f52a3d41cd5d28d6058a31b6d2b790044aa49252/msteams-platform/apps-in-teams-meetings/API-references.md) | 92 | 23 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5d8b3bbe-be55-46d1-9697-2e764c6183f9) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f52a3d41cd5d28d6058a31b6d2b790044aa49252/msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d447582a-b0d5-4ed9-9a7f-c71af6273f46) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c79ae69a20b8e47ded87ff31781dad4b1f93540e/msteams-platform/apps-in-teams-meetings/API-references.md) | 92 | 23 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e7e217d0-3b85-404c-ad1f-7bf3ad2c40c5) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c79ae69a20b8e47ded87ff31781dad4b1f93540e/msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b87b5e47-4b21-4aa9-a5cf-f1d090877bf4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d39d094f9c3870d3f71a4dccc6e0d63545fbe931/msteams-platform/apps-in-teams-meetings/API-references.md) | 92 | 23 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eae20f8d-85f6-4244-94f6-94f82ca6d27c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d39d094f9c3870d3f71a4dccc6e0d63545fbe931/msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d63eaa69-205d-427b-a080-071b47cb1dad) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0743e686260ebe07a4e7203ff68f9d50e1adafb5/msteams-platform/apps-in-teams-meetings/API-references.md) | 92 | 23 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0c59bda8-7c52-4ec3-9fc1-5218924ae583) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0743e686260ebe07a4e7203ff68f9d50e1adafb5/msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/464ea303-bf1c-4eb7-b36e-3c619cdcaca0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by arocc — APPROVED** [review]
Thank you for the updates, looks great!

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c5e795419f25a9386af3f3aba458517f1f8d5284/msteams-platform/apps-in-teams-meetings/API-references.md) | 92 | 23 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/52ac1257-b858-485d-8e45-8b8ad9b1c589) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c5e795419f25a9386af3f3aba458517f1f8d5284/msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/babe32df-555f-4457-8772-5d6662565cb4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/52a7047c28e3a0270862f25dc7a0872ae8f3fe34/msteams-platform/apps-in-teams-meetings/API-references.md) | 92 | 24 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b8711088-c23c-41bc-b33e-f062f0db5aa0) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/52a7047c28e3a0270862f25dc7a0872ae8f3fe34/msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/876ff6ce-93c6-487d-a51b-eaccb3497434) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9ad3c32a47ae49a77bb3d8306514275237d1ef84/msteams-platform/apps-in-teams-meetings/API-references.md) | 92 | 24 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/05178cd3-abfe-495c-8eb5-a4776efb7efe) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9ad3c32a47ae49a77bb3d8306514275237d1ef84/msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d8a2c485-1bb0-4cd6-bcd3-fbcf8e169228) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3bcc7641fdd7e0c216e9852c6f7b40f842e0e7ab/msteams-platform/apps-in-teams-meetings/API-references.md) | 92 | 24 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/44784730-65e0-499c-81a0-85a3bf969166) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3bcc7641fdd7e0c216e9852c6f7b40f842e0e7ab/msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0ec57839-293a-4dd6-b009-a1869c77b2a0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — APPROVED** [review]
Approved

**Pull request merged**